broker/cloud_run/lsst/ps_to_storage/main.py

```diff
@@ -95,148 +84,62 @@ def run():
 def store_alert_data(envelope) -> None:
```

Comment:
Let's get rid of this as a separate function. What's left is simple enough to go directly in `run`.